examples/duet/vae/AutoEncoder_Syft_Data_Scientist.ipynb

```diff
@@ -1,8 +1,395 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">dry_run = False
epochs = 10
config = {
&nbsp; &nbsp; "batch_size":128,
&nbsp; &nbsp; "no_cuda":True,
&nbsp; &nbsp; "seed":42,
&nbsp; &nbsp; "epochs":epochs,
&nbsp; &nbsp; "dry_run":dry_run,
&nbsp; &nbsp; "log_interval":10,
}
</pre><p>If we put some variables on single lines which we want to be able to change depending on CI we can easily use the new integration test line replacer to change dry_run to True and epochs to 1 so that users get a version they want to run vs one we run in CI. In this case because of all the dry_run breaks we can probably use dry_run=True so that users can run the notebook quickly as well and then adjust.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5299/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Done